client/lcd/lcd_test.go

```diff
@@ -294,6 +295,7 @@ func TestTxs(t *testing.T) {
 // strt TM and the LCD in process, listening on their respective sockets
 func startTMAndLCD() (*nm.Node, net.Listener, error) {
 
+	viper.Set(cli.HomeFlag, os.TempDir())
```

Comment:
++

Reply:
Do you mean this should be changed to something else?

The keybase singleton reads the home directory; this just ensures `make test` will put data in `/tmp/.lcd` on Unix, for example, instead of using the user's actual home directory.

Reply:
++ means <3 

this is perfect for now.

Though I think we've seen `os.TempDir()` fail in cases where /tmp had different permissions or something. 

In an ideal world (I think) we'd be able to run these tests fully in memory, but we're not there yet!